src/renderer/agents/help_agent.ts

```
@@ -0,0 +1,364 @@
+    ["viewer", ["viewer", "mulmo", "視聴", "再生"]],
+    ["dev", ["dev", "開発", "yarn", "start"]],
+    ["all", ["all", "一覧", "目次"]],
+  ];
+  for (const [t, kws] of table) {
+    if (kws.some((kw) => key.includes(kw))) return t;
+  }
+  return fallback;
+}
+
+// --- agent ---
+export const mulmoCastHelpAgent: AgentFunction = async ({ params, namedInputs }) => {
```

ESLint

[error] 'AgentFunction' is not defined.

(no-undef)